Client/Frontend/Strings.swift

```diff
@@ -1069,8 +1069,8 @@ extension String {
             public static let Description = MZLocalizedString(
                 key: "Onboarding.Sync.Description.v120",
                 tableName: "Onboarding",
-                value: "When you’re signed in and synced, you’re safer. Firefox encrypts your passwords, bookmarks, and more.",
-                comment: "String used to describes the description of what Firefox is on the Sync onboarding page for current version in our Onboarding screens.")
+                value: "When you’re signed in and synced, you’re safer. %@ encrypts your passwords, bookmarks, and more.",
```

Comment:
Given that we've updated this to use a placeholder, have we also updated the place in the app where it's used to fill in the placeholder?

Reply:
the string is not currently used in the app